config/biome.jsonc

```
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://biomejs.dev/schemas/2.0.0-beta.5/schema.json",
+	"vcs": {
+		"enabled": true,
+		"clientKind": "git",
+		"useIgnoreFile": true
+	},
+	"files": {
+		"ignoreUnknown": true,
+		"includes": [
```

Biome (1.9.4)

[error] Found an unknown key `includes`.

Known keys:


(deserialize)